docs/prologue/introduction.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Introduction"
+description: "The Tempo Operator provides Kubernetes native deployment and management of Tenoi and related tracing components"
+lead: ""
+date: 2022-06-21T08:48:45+00:00
+lastmod: 2022-06-21T08:48:45+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "prologue"
+weight: 100
+toc: true
+---
+
+The Tempo Operator provides [Kubernetes](https://kubernetes.io/) native deployment and management of [Tempo](https://github.com/grafana/tempo) and related tracing components.
+The purpose of this project is to simplify and automate the configuration of a Tempo based tracing stack for Kubernetes clusters.
+
+The Tempo operator includes, but is not limited to, the following features:
+
+* Kubernetes Custom Resources: Use Kubernetes custom resources to deploy and manage Tempo, and related components.
+* Simplified Deployment Configuration: Configure the fundamentals of Tempo like tenants, limits, replication factor and storage from a native Kubernetes resource.
+
+## Get started
```

Comment:
I understand we don't need to cover everything as part of this PR but, how about creating an issue to ensure we don't forget about this?

Reply:
Created: https://github.com/os-observability/tempo-operator/issues/275